### PR TITLE
Mjw/yaml build

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  buildConfiguration: 'release'
+  buildPlatform: 'anycpu'
+
+pool:
+  name: 'Azure Pipelines'
+  vmImage: 'ubuntu-16.04'
+
+steps:
+
+- task: gittools.gitversion.gitversion-task.GitVersion@5
```
Comment (chelnak) [inline azure-pipelines.yml]: Should we be using the preview here?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+name: TL-Marketing-and-Communications
```
Comment (jack-education) [inline azure-pipelines.yml]:
```suggestion
name: tl-marketing-and-communications
```

Comment (jack-education) [reply azure-pipelines.yml]: Convention is normally lowercase

File: azure-pipelines.yml (hunk 1 above)
Comment (mwevill) [reply azure-pipelines.yml]: it works

Comment (EwanNoble) [reply azure-pipelines.yml]: it's GA now

File: azure-pipelines.yml (hunk 2 above)
Comment (EwanNoble) [reply azure-pipelines.yml]: That name field is build number format, so this is incorrect either way, it'll be set by Gitversion

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
+name: tl-marketing-and-communications
```
Comment (EwanNoble) [inline azure-pipelines.yml]: This is build number format set by Gitversion, it isn't needed here

File: azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    command: publish
+    publishWebProjects: True
+    arguments: '--configuration $(buildConfiguration) --output $(build.artifactstagingdirectory)'
+    zipAfterPublish: True
+
+- task: CopyFiles@2
+  displayName: 'Copy Files to: $(build.artifactstagingdirectory)'
+  inputs:
+    Contents: |
+     azure\**
```
Comment (EwanNoble) [inline azure-pipelines.yml]: If you're using Ubuntu you need forward slashes 
```suggestion
     azure/**
```

**Pull request merged**